src/dali_executor/dali_pipeline.h

```diff
@@ -110,10 +110,14 @@ class DaliPipeline {
       dali_data_type_t data_type, span<const int64_t> inputs_shapes,
       int sample_ndims);
 
+  void SetInput(
+      const void* ptr, const char* name, device_type_t source_device,
+      dali_data_type_t data_type, TensorListShape<> input_shape);
+
   void PutOutput(
       void* destination, int output_idx, device_type_t destination_device);
 
-  const int batch_size;
+  const int max_batch_size;
```

Comment:
Since you're changing it already, could you change it to `const in kMaxBatchSize` or `int max_batch_size_`?

Reply:
renamed to max_batch_size_